src/atoms/tile/AppTile.ts

```diff
@@ -0,0 +1,38 @@
+import { TTile } from '../../types';
+
+export function AppTile({
+    height,
+    width = 'w-full',
+    textContent,
+    textSize,
+    imgUrl,
+    imgOptions,
+    onClick,
+}: TTile): HTMLElement {
+    const tile = document.createElement('div');
+    tile.className = `rounded-lg bg-tile-bg-gray flex items-center`;
+    tile.classList.add(width, height);
```

Comment:
13-14 rows could be optimized in one:
tile.classList.add('rounded-lg', 'bg-tile-bg-gray', 'flex', 'items-center', width, height);

Reply:
Fixed